statsmodels/multivariate/multivariate_ols.py

```diff
@@ -20,22 +20,29 @@
 _hypotheses_doc = """
     hypotheses: A list of tuples
         Hypothesis to be tested. Each element is an array-like
-                            (name, contrast_L, transform_M)
-        containing a string `name`, the contrast matrix L and the transform
-        matrix M (for transforming dependent variables), respectively.
+                       (name, contrast_L, transform_M)
+                                    or
+                       (name, contrast_L, transform_M, constant_C)
+        containing a string `name`, the contrast matrix L, the transform
+        matrix M (for transforming dependent variables), and right-hand side
+        constant matrix constant_C, respectively.
         contrast_L : array-like or an array of strings
-           Contrast matrix for hypotheses testing.
+           Left-hand side contrast matrix for hypotheses testing.
            If array-like, each row is an hypotheses and each column is an
            independent variable. At least 1 row
            (1 by k_exog, the number of independent variables) is required.
            If an array of strings, it will be passed to
            patsy.DesignInfo().linear_constraint.
         transform_M : array-like or an array of strings
-            Transform matrix.
+            Left hand side transform matrix.
             Default to be k_endog by k_endog identity matrix
             (i.e. do not transform y matrix).
-           If an array of strings, it will be passed to
-           patsy.DesignInfo().linear_constraint.
+            If an array of strings, it will be passed to
+            patsy.DesignInfo().linear_constraint.
+        constant_C : array-like
```

Comment:
add, "optional"

Should we also make transform_M optional, i.e. also allow `len(hypothesis) == 2`?
But it cannot be skipped if constant_C is specified, to make it unambiguous.